### DISCUSSION TITLE
Feedback/orientação de professionais mais experiêntes.

### DISCUSSION THREAD

**Discussion by devlucaslopes** [frontendbr/forum#1995] [Discussão]
Eai pessoal, tudo bem? 

Meu nome é Lucas, tenho 23 anos e trabalho como dev há 4 anos. A maior parte das minhas experiências profissionais foram em startups no estágio inicial, porque sempre curti essa vibe de tirar as ideias do papel e ver a empresa tomando forma. 

Mas de um tempo pra cá assumi um papel maior dentro da empresa e me veio aquela famosa síndrome do impostor. Será que eu to indo pelo caminho certo? Não sei se eu vou conseguir fazer...esse tipo de insegurança começou a me atrapalhar um pouco. Conversei com o pessoal da empresa e pedi para trazer uma pessoa com mais experiência para ter alguem para conversar e discurtir sobre as soluções do dia-a-dia. 

Durante esses 4 anos eu trabalhei com back-end e front-end, mas no final do ano passado eu resolvi me dedicar ao front porque me identificava mais e surgiram algumas demandas que me levaram para esse lado. 

Tenho um pouco de insegurança por ter uma trajetória profissional meio solitária...Gostaria de conversar com pessoas mais experiêntes sobre o universo do front-end e pegar algumas dicas/orientações para seguir nesse caminho do front ;)

**Comment by mitestainer** [comment]
Cara, vc definiu 100% o meu momento. Tô na mesma vibe de dúvidas e insegurança haha vou ficar de olho nessa thread 👀

**Comment by thyagorafaell** [comment]
Ainda não usei, mas recentemente descobi o [Mentora](https://mentora.webflow.io).

Tem um pessoal experiente lá, e diversas categorias (carreira, liderança, front, back, ux...)

**Comment by fabiochiquezi** [comment]
Acho que esse sentimento quase nunca vai embora haha, tbm tenho uma trajetória mais solitária e é comum sempre pegar um desafio e ter aquela sensação "será que consigo?", mas depois de alguns desafios vencidos você vai ganhando confiança e indo na cara e coragem kk

**Reply by leandrocrs** [reply]
Relaxa, é da idade. Quero dizer, a sensação vai continuar batendo sempre que for um desafio grande, mas vai te afetar cada vez menos na vida conforme for ficando mais experiente.

O importante, se tratando de cargos de liderança e gestão é não acreditar que você deve ser o dono da verdade ou o especialista master porque isso simplesmente não funciona dessa forma.

Faça sempre ser um processo coletivo, trazendo os liderados junto consigo. Isso te fará um bom líder/gestor, e ainda te munirá de todas as informações que você precisará, através do seu time.

Se for pra uma posição solo, não se preocupe, vá com frio na barriga mesmo e espere 3 semanas. Geralmente essa sensação passa na primeira entrega.
